test/integration/controllers/message.spec.js

```diff
@@ -0,0 +1,62 @@
+const { serverInit, serverCleanup, stopServer } = require('~/test/setup')
+const testUserAuthentication = require('~/utils/testUserAuth')
+const { expectError } = require('~/test/helpers')
+const { UNAUTHORIZED } = require('~/consts/errors')
+
+const endpointUrl = '/messages/'
+
+let accessToken
+let messageData
+
+let messageBody = {
+  text: 'SOme amount of text',
+  isRead: false,
+  isNotified: false,
+  chat: '6421d9833cdf38b706756dfd'
+}
+
+describe('Message controller', () => {
+  let app, server
+
+  beforeAll(async () => {
+    ;({ app, server } = await serverInit())
+  })
+
+  beforeEach(async () => {
+    accessToken = await testUserAuthentication(app)
+  })
+
+  afterEach(async () => {
+    await serverCleanup()
+  })
+
+  afterAll(async () => {
+    await stopServer(server)
+  })
+
+  describe(`POST ${endpointUrl}`, () => {
+    it('should throw UNAUTHORIZED', async () => {
+      const response = await app.post(endpointUrl).send(messageBody)
+
+      expectError(401, UNAUTHORIZED, response)
+    })
+
+    it('should create a new message', async () => {
+      const testMessage = await app.post(endpointUrl).set('Authorization', `Bearer ${accessToken}`).send(messageBody)
+
+      expect(testMessage.statusCode).toBe(201)
+
+      messageData = {
```

Comment:
move it to the top, where all variables are

Reply:
Fixed.